reports.js

```diff
@@ -38,7 +38,20 @@ async function pluginsReport() {
     report.push(plugin);
   });
   report.sort((a, b) => b.installs - a.installs);
-  return report;
+
+  const reducer = (accumulator, currentValue) => accumulator + currentValue;
+
+  const todo = report.map(plugin => plugin.status === 'TODO' ? 1 : 0).reduce(reducer);
```

Comment:
So not that it probably matters for the data set this small, but these 3 lines loop through the entire data set 6 times.

I think report.filter().length is better (one loop per status), or reduce the entire data into an object keyed by status (loop one) then use those values as the length

Reply:
reduced to just one loop